mail-client/alot/alot-0.9.1.ebuild

```diff
@@ -18,17 +19,16 @@ IUSE="doc test"
 RESTRICT="!test? ( test )"
 
 RDEPEND="
-	>=app-crypt/gpgme-1.9.0[python,${PYTHON_USEDEP}]
-	>=dev-python/configobj-4.7.0[${PYTHON_USEDEP}]
+	app-crypt/gpgme[python,${PYTHON_USEDEP}]
```

Comment:
why did you remove `>=`?

Reply:
Just because the minimal available version in portage is sufficient (1.13.0).